tests/group_functions/CMakeLists.txt

```diff
@@ -1,17 +1,17 @@
-function(configure_test_case)
-    cmake_parse_arguments(CTS
-	    "" "TYPE;IN_FILENAME;OUT_FILENAME;TEST_LIST" "" ${ARGN})
-    set(CTS_TYPE_NAME ${CTS_TYPE})
-    configure_file(${CTS_IN_FILENAME} ${CTS_OUT_FILENAME})
-    list(APPEND ${CTS_TEST_LIST} "${CMAKE_CURRENT_BINARY_DIR}/${CTS_OUT_FILENAME}")
-    set(${CTS_TEST_LIST} ${${CTS_TEST_LIST}} PARENT_SCOPE)
+function(configure_test_case)    
```

Comment:
Also configure your tools and GitHub to flag/remove trailing spaces. :-)

Reply:
@keryell Done. Thanks for informing! I have updated vscode config as well